### PR TITLE
Created script action 111: Un-register Team success

### PR DESCRIPTION
**Pull request by FS-21**
Is just the opposite effect of 49,0

### CODE REVIEW

File: src/Ext/Script/Body.cpp [hunk 1]
```diff
@@ -40,6 +40,10 @@ void ScriptExt::ProcessAction(TeamClass* pTeam)
 	case 73:
 		ScriptExt::WaitUntillFullAmmoAction(pTeam);
 		break;
+	case 74: // IMPORTANT: This action will be renamed from 74 to 111 or another number when PR 296 gets merged into Develop branch
+		// Un-register success for AITrigger weight adjustment (this is the opposite of 49,0)
+		ScriptExt::UnregisterGreatSuccess(pTeam);
+		break;
```
Comment (Metadorius) [inline src/Ext/Script/Body.cpp:46]: We shouldn't merge it like so, make it 111 or the number that you will implement without the need for other changes further.

**Review by Metadorius — CHANGES_REQUESTED** [review]
Changing script action numbers further is a very bad idea, change it to a final or at least semi-final number (111 I guess?)

**Comment by Metadorius** [comment]
Also no changelog mention.

File: src/Ext/Script/Body.cpp (hunk 1 above)
Comment (FS-21) [reply src/Ext/Script/Body.cpp:46]: The problem is FA2 that script action numbers should be consecutive. The last one was 73 and the 74 and forward are the ones of PR #296, being the 111 the first free slot for any of the new script actions I'm creating.

Comment (FS-21) [reply src/Ext/Script/Body.cpp:46]: Ok, I leave the definitive number 111 and made the mention f the action.

**Pull request closed without merging**